### PR TITLE
Add a command to fetch the Go tree for harvey

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: .github/workflows/harvey.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y rc qemu go-dep qemu-utils expect qemu-system-x86
         go get harvey-os.org/cmd/ufs
+    - name: install source we build into the image
+        git clone --depth 1 --branch go1.16.6 git@github.com:golang/go.git
```
Comment (gmacd) [inline .github/workflows/harvey.yml:20]: Should this be under a ‘run’ command like the others?

**Review by 0intro — APPROVED** [review]

File: .github/workflows/harvey.yml (hunk 1 above)
Comment (rminnich) [reply .github/workflows/harvey.yml:21]: Thanks, let's see what that does.

**Review by gmacd — APPROVED** [review]

**Review by gmacd — DISMISSED** [review]

**Comment by rminnich** [comment]
well, this does seem to work, even if the commands all end up in /go ...

File: .github/workflows/harvey.yml [hunk 2]
```diff
@@ -16,6 +16,16 @@ jobs:
...
+    - name: install source we build into the image
+      run: |
+        git clone --depth 1 --branch go1.16.6 https://github.com/golang/go.git
+        export GOBIN=`pwd`/amd64/go/bin
+        echo $GOBIN
+        mkdir -p $GOBIN
+        ls -R $GOBIN
+        cd go/src
+        GOOS=plan9 GOARCH=amd64 ./make.bash
+        ls -R $GOBIN
```
Comment (orangecms) [inline .github/workflows/harvey.yml:28]: To verify, a little `file $GOPATH/...` could do.

**Review by floren — APPROVED** [review]

File: .github/workflows/harvey.yml (hunk 2 above)
Comment (rminnich) [reply .github/workflows/harvey.yml:28]: the file command seems to view plan 9 binaries as 'data'

**Pull request merged**